ui/fisher/fisher-ui.js

```diff
@@ -16,7 +18,7 @@ if (window.customElements) {
   });
 }
 
-class FisherUI {
+export default class FisherUI {
```

Comment:
fisher-ui.js uses `Options` but it is defined in the other file.  I think if you pass it to the constructor and use `this.options` in the two places below, fisher will work properly again.